native/src/test/scala/org/portablescala/reflect/TestPlatform.scala

```diff
@@ -0,0 +1,5 @@
+package org.portablescala.reflect
+
+object TestPlatform {
+  val isScala210OnJVM = false
```

Comment:
Scala 2.10 has been declared dead. This file shouldn't exist :wink: 
https://github.com/portable-scala/portable-scala-reflect/pull/29#issuecomment-766281022

Reply:
Understood.  A WIP artifact that I did not expect to see in this workspace now.
I will make sure it is gone in the final PR.

Thanks,